tests/src/Kernel/SiteBrandingBlockTest.php

```diff
@@ -75,8 +75,6 @@ public function testBlockRendering(): void {
 
     $actual = $crawler->filter('.site-name');
     $this->assertCount(1, $actual);
-    $link = $actual->filter('.h1.text-white.text-decoration-none.align-bottom');
```

Comment:
put this back once you put back the link, and I think we have new css classes to check against